### ISSUE TITLE
Using pengwin-setup to install just Python fails unless you also add C++

### ISSUE THREAD

**Issue by andybak** [WhitewaterFoundry/Pengwin#657]
**Describe the bug**
Using pengwin-setup to install just Python fails unless you also install C++

**To Reproduce**
1. Install Pengwin
2. Choose just "Python" in the GUI setup
3. Type python in the shell
4. You get an error (sorry - didn't note down the exact wording but it was basically "install failed - no C compile found")

**Expected behavior**
Python works. 

**What other troubleshooting have you attempted?**

I reran setup and chose the C++ tools as well. It worked.

**Pengwin Version**

Run `grep "PENGWIN_VERSION" /etc/os-release` in Pengwin.

PENGWIN_VERSION="20.11.0"

**WSL general info**

Run `wslsys` in Pengwin.

WSL Version: 2
Locale: en_GB
Release Install Date: Fri Aug 28 01:11:36 BST 2020
Branch: 19h1_release
Build: 18363
Full Build: 18362.1.amd64fre.19h1_release.190318-1202
Display Scaling: 1.25
Windows Theme: light
Windows Uptime: 17d 23h 35m
WSL Uptime: 1d 9h 21m
WSL Release: Pengwin
WSL Kernel: Linux 4.19.128-microsoft-standard
Packages Count: 524

**Comment by WSLUser** [comment]
Out of curiosity, did you also try adding anything besides just Python (such as Poetry or Pip?)

**Comment by andybak** [comment]
> Out of curiosity, did you also try adding anything besides just Python (such as Poetry or Pip?)

I don't think so. I've got another machine I can try this on in a more scientific way if that helps but if I recall correctly the pengwin-setup menu only has a single entry for each language. I don't recall seeing options for adding indivual setup tools such as Poetry.

**Comment by WSLUser** [comment]
It would be when you select Python. An option should appear for you to choose to install Pip or Poetry with the IDLE Manager. If these aren't showing up, I need to submit a fix (I added Poetry as an alternative to pip in pengwin-setup).

**Comment by WSLUser** [comment]
I submitted a minor PR that may help ensure you see the options. But I don't think it would resolve your issue. Poetry doesn't require C++, it's purely written in python so I think that is not the cause either. Carlos will need to figure out what's causing this odd dependency to show up (perhaps something in another script is affecting the pythonpi script?)

**Comment by crramirez** [comment]
Hello,

I have been making an intensive test and the behavior doesn't happen in my environment. That is installed from scratch.

Probably it was an upstream issue in Debian repos because you can see a mix of Python 3.8 and 3.9 packages when you install.

I am preparing an update to properly show a many asking how you want python to be installed and also upgraded to 3.9

Regards,
Carlos

**Comment by WSLUser** [comment]
I can confirm this issue does not seem to be observed in a fresh install. @andybak can you retest? If you don't reproduce after updating or trying a fresh install of Pengwin, please close this issue.
